docs/api-testing/integrations/pagerduty-webhooks.md

```diff
@@ -67,6 +67,25 @@ Like the PagerDuty Connector, setting up an outgoing webbook connector allows Sa
 ### Incoming Webhooks
 See [Adding Incoming Webhooks](/api-testing/integrations/apifctl-cicd-integration/#adding-incoming-webhooks).
 
+## Testing Connectors
+The Test Connectors feature tests the functionality of each connector as defined in its settings, but will not display any notifications on the dashboard. Testing a connector before executing it can minimize the time you spend creating connectors.
+
+To test a connector:
```

Comment:
@lauramthomas This feature tests all the connectors you have set up with one click so it is better to use the plural

Reply:
Fixed